packages/webapp/src/components/Modals/QuickAssignModal/index.jsx

```diff
@@ -126,7 +83,7 @@ export default function TaskQuickAssignModal({
           assignee_user_id: assigneeUserId,
         });
 
-    if (isYesOptionSelected(selectedHourlyWageAction)) {
+    if (shouldSetWage) {
       const wage = +parseFloat(hourlyWage).toFixed(2);
```

Comment:
if {valueAsNumber: true} is passed as option in hourlywage register, HOURLY_WAGE will be treated as a number by  react-hook-form and the value won't have be to parsed to number manually